_FIPS201/requirements.md

```diff
@@ -76,14 +76,14 @@ For full guidance on PIV credentialing investigative and adjudicative requiremen
 
 ## 2.3 Biometric Data Collection for Background Investigations {#s-2-3}
 
-A full set of fingerprints **SHALL** be collected from each PIV applicant. 
+This section applies to applicant that **DO NOT** have a completed and favorably adjudicated Tier 1 or higher federal background investigation on record that can be located and referenced.
+
+A full set of fingerprints **SHALL** be collected from each PIV applicant who is lacking on-record background investigation. 
```

Comment:
The added words seem redundant. I'm wondering about this entire section: since Section 2.2 already requires the NCHC to be completed and favorably adjudicated, would these requirements ever come into effect? The important thing is that the fingerprints on the PIV card correlate with those collected for NCHC.